src/gwt/src/org/rstudio/studio/client/workbench/views/environment/dataimport/DataImport.java

```diff
@@ -33,28 +44,58 @@
    private static DataImportUiBinder uiBinder = GWT
          .create(DataImportUiBinder.class);
    
+   private final SourceServerOperations server_;
+   private final int maxRows_ = 200;
+   
    interface DataImportUiBinder extends UiBinder<Widget, DataImport>
    {
    }
 
-   public DataImport()
+   public DataImport(SourceServerOperations server)
```

Comment:
Dependencies like this would typically be satisfied using an `@inject void initialize(..)` method. Easiest if @jmcphers shows you how this works in person.